### ISSUE TITLE
Implement BNTT (Batch Norm through time)

### ISSUE THREAD

**Issue by cpehle** [norse/norse#345]
This appears to be an effective method to train larger scale convolutional spiking neural networks. The reference implementation can be found here: https://github.com/Intelligent-Computing-Lab-Yale/BNTT-Batch-Normalization-Through-Time. Effectively this just maintains a batch norm function for each timestep. I therefore suggest to implement an abstraction which allows us to "replicate" any kind of point wise applicable torch layer across the time dimension with independent parameters for each time step.
